dev/TabView/TabView.cpp

```diff
@@ -227,6 +237,20 @@ void TabView::OnSelectedItemPropertyChanged(const winrt::DependencyPropertyChang
     UpdateSelectedItem();
 }
 
+void TabView::OnTabStripFooterPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
+{
+    if (const auto oldFooter = args.OldValue().try_as<winrt::FrameworkElement>())
+    {
+        oldFooter.UnregisterPropertyChangedCallback(winrt::FrameworkElement::MinWidthProperty(), m_footerMinWidthProperyChangedToken);
+        oldFooter.UnregisterPropertyChangedCallback(winrt::FrameworkElement::WidthProperty(), m_footerWidthProperyChangedToken);
+    }
+    if (const auto newFooter = args.NewValue().try_as<winrt::FrameworkElement>())
+    {
+        m_footerMinWidthProperyChangedToken = newFooter.RegisterPropertyChangedCallback(winrt::FrameworkElement::MinWidthProperty(), { this,&TabView::OnComponentSizeChanged });
```

Comment:
>OnComponentSizeChanged [](start = 150, length = 22)

nit: Component sounds very generic. perhaps OnFooterSizeChanged ?

Reply:
Good idea, changed the name to OnFooterSizeChanged.